linux/apt_packages.nice_to_have.txt

```diff
@@ -13,6 +13,7 @@ p7zip-full                  # 7z and 7za file archivers with high compression ra
 p7zip-rar                   # non-free rar module for p7zip
 pv                          # Shell pipeline element to meter data passing through
 ssh-askpass                 # under X, asks user for a passphrase for ssh-add
+sshpass                     # non-interactive ssh password auth
```

Comment:
I'm ok with sshpass, but typically its blocked by security. If John is ok with it, I'm too.